streaming-binlog/src/main/java/org/openmrs/analytics/FhirConverter.java

```diff
@@ -92,8 +101,46 @@ public void process(Exchange exchange) {
 			return;
 		}
 		if (payload.get("uuid") == null) {
-			log.error(String.format("No uuid column for table %s ignoring payload %s ", table, payload));
-			return;
+			Connection conn = null;
+			Statement stmt = null;
+			final String USER = "root";
+			final String PASS = "debezium";
+			String sqlStmt = null;
+			try {
+				Class.forName("com.mysql.jdbc.Driver");
+				
+				//Connecting to openmrs database
+				conn = DriverManager.getConnection(DB_URL, USER, PASS);
+				
+				//Creating statement
+				stmt = conn.createStatement();
+				String sql = config.getJoinClause();
+				ResultSet rs = stmt.executeQuery(sql);
```

Comment:
@gitcliff   ideally the plain join join clause ie `location.location_id = location_tag_map.location_id"`  can not be executed as an sql statement . 
.simply use a parametrised slq query like
 ```
 SELECT uuid
FROM person
WHERE person.person_id = {patient_id}

and pass in the patient id .
```

You can make it more generic like 

SELECT uuid
FROM {parent_table}
WHERE  {join_clause}

and join clause can be as simple as